project-base/storefront/pages/customer/users/index.tsx

```diff
@@ -0,0 +1,146 @@
+    const [, removeCustomerUser] = useRemoveCustomerUserMutation();
+    const [customerUsersUrl, customerUserAddNew, customerUserEditUrlByUuid] = getInternationalizedStaticUrls(
+        ['/customer/users', '/customer/users/add', { url: '/customer/users/:uuid', param: '' }],
+        url,
+    );
+    const breadcrumbs: TypeBreadcrumbFragment[] = [
+        { __typename: 'Link', name: t('Customer users'), slug: customerUsersUrl },
+    ];
+    const currentCustomerUsers = useCurrentCustomerUsers();
+    const gtmStaticPageViewEvent = useGtmStaticPageViewEvent(GtmPageType.other, breadcrumbs);
+    useGtmPageViewEvent(gtmStaticPageViewEvent);
+
+    const deleteItemHandler = async (customerUserUuid: string | undefined) => {
+        if (customerUserUuid === undefined) {
+            return;
+        }
+
+        updatePortalContent(null);
+        const deleteCustomerUserResult = await removeCustomerUser({ customerUserUuid });
+
+        if (deleteCustomerUserResult.error !== undefined) {
+            showErrorMessage(
+                deleteCustomerUserResult.error.message
+                    ? deleteCustomerUserResult.error.message
+                    : t('There was an error while deleting user'),
+                GtmMessageOriginType.other,
+            );
+            return;
+        }
+
+        showSuccessMessage(t('User has been deleted'));
+    };
+
+    const openDeleteCustomerUserPopup = (
+        e: React.MouseEvent<HTMLButtonElement, MouseEvent>,
+        customerUserToBeDeletedUuid: string,
+    ) => {
+        e.stopPropagation();
+        updatePortalContent(
+            <DeleteCustomerUserPopup
+                deleteCustomerUserHandler={() => deleteItemHandler(customerUserToBeDeletedUuid)}
+            />,
+        );
+    };
+
+    return (
+        <>
+            <MetaRobots content="noindex" />
+            <CustomerLayout breadcrumbs={breadcrumbs} pageHeading={t('Customer users')} title={t('Customer users')}>
+                <div className="w-full flex flex-col gap-4">
+                    <ExtendedNextLink href={customerUserAddNew}>
+                        <Button className="w-fit" size="small" variant="inverted">
+                            {t('Add new user')}
+                        </Button>
+                    </ExtendedNextLink>
+                    <Table className="w-full border-0 p-0">
+                        {currentCustomerUsers.map((user) => (
+                            <Row
+                                key={user.uuid}
+                                className="bg-tableBackground odd:bg-tableBackgroundContrast border-none"
+                            >
+                                <Cell className="py-2 text-left text-sm font-bold uppercase leading-5">
+                                    {user.firstName} {user.lastName}
+                                </Cell>
+
+                                <Cell className="py-2 text-left text-sm leading-5 hidden vl:table-cell">
```

Comment:
I suggest checking if it’s okay to hide the email on small devices. Alternatively, we could consider displaying it under the name for better orientation.

Reply:
I will write comment about it and will discuss during BV.